interceptlogger.go

```diff
@@ -213,18 +213,34 @@ func (i *interceptLogger) DeregisterSink(sink SinkAdapter) {
 // Create a *log.Logger that will send it's data through this Logger. This
 // allows packages that expect to be using the standard library to log to
 // actually use this logger, which will also send to any registered sinks.
-func (l *interceptLogger) StandardLoggerIntercept(opts *StandardLoggerOptions) *log.Logger {
+func (i *interceptLogger) StandardLoggerIntercept(opts *StandardLoggerOptions) *log.Logger {
 	if opts == nil {
 		opts = &StandardLoggerOptions{}
 	}
 
-	return log.New(l.StandardWriterIntercept(opts), "", 0)
+	return log.New(i.StandardWriterIntercept(opts), "", 0)
 }
 
-func (l *interceptLogger) StandardWriterIntercept(opts *StandardLoggerOptions) io.Writer {
+func (i *interceptLogger) StandardWriterIntercept(opts *StandardLoggerOptions) io.Writer {
 	return &stdlogAdapter{
-		log:         l,
```

Comment:
Apologies for the pedantry with these variable renames, but my editor was complaining that they weren't consistent throughout the file.

Reply:
All good! They should be `i` anyway.